src/Pages/Category/Category.scss

```
@@ -0,0 +1,244 @@
+@import '../../Styles/variables.scss';
+
+.Category {
+  color: #333;
+  .banner {
+    position: relative;
+    height: 390px;
+    background-color: #dbf4f9;
+    z-index: -1;
+
+    .arrow {
+    }
+
+    .caption {
+      position: absolute;
+      top: 50%;
+      left: 10%;
+      transform: translateY(-50%);
+
+      .subtitle {
+        font-size: 20px;
+        margin-bottom: 3px;
+      }
+
+      .title {
+        position: relative;
+        font-size: 55px;
+        font-weight: bold;
+        margin-bottom: 45px;
+
+        &::after {
+          content: '';
+          position: relative;
+          top: 30px;
+          display: block;
+          width: 26px;
+          height: 3px;
+          background-color: #333;
+        }
+      }
+
+      .desc {
+        font-size: 18px;
+        margin-bottom: 15px;
+        line-height: 25px;
+      }
+    }
+  }
+  .categoryBody {
+    .productFilter {
+      margin: 55px 0;
+
+      ul {
+        text-align: center;
+      }
+
+      li {
+        display: inline-block;
+        padding: 15px 30px;
+        margin: 0 7px;
+        border-radius: 2rem;
+        font-size: $main-font-size;
+        color: #c9c9c9;
+        background-color: $category-background-color;
+
+        &:hover {
+          color: #333;
+          background-color: $yellow;
+          cursor: pointer;
+        }
+      }
+
+      .active {
+        color: #333;
+        background-color: $yellow;
+      }
+    }
+
+    .productLnb {
+      display: flex;
+      justify-content: space-between;
+      width: 1166px;
+      margin: 0 auto;
+      margin-bottom: 45px;
+      font-size: $main-font-size;
+
+      .productNum {
+        color: #333;
+      }
+
+      .listSort {
+        position: relative;
+        padding-right: 20px;
+        font-size: 16px;
+        color: #6a6666;
+
+        .sortArrow {
+          position: absolute;
+          right: 0;
+          content: '';
+          display: block;
+          width: 8px;
+          height: 8px;
+          transform: rotate(-45deg);
+        }
+
+        .unClicked {
+          top: 6px;
+          border-left: 1px solid #827487;
+          border-bottom: 1px solid #827487;
+        }
+
+        .clicked {
+          top: 11px;
+          border-top: 1px solid #827487;
+          border-right: 1px solid #827487;
+        }
+
+        .sortPopup {
+          position: absolute;
+          top: 50px;
+          padding: 18px 0 20px;
+          right: -15px;
+          width: 143px;
+          height: 200px;
```

```suggestion
        .sortPopup {
          position: absolute;
          top: 50px;
          right: -15px;
          padding: 18px 0 20px;
          width: 143px;
          height: 200px;
```
읽을 때 혼동 없도록 속성들은 연관된 것끼리 묶어 표기해주면 좋습니다.

---

네 묶어서 표시하겠습니다